### PR TITLE
RPS v1

### PR DESCRIPTION
**Pull request by vaelentine**
Rock Paper Scissors

### CODE REVIEW

File: mags/RPS.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+# For storing all of the game logic and methods
+class RPS:
+    name = "Rock, Paper, Scissors"
+    choices = ["rock", "paper", "scissors"]
+    win = [0, -1, 1], [1, 0, -1], [-1, 1, 0]
+    outcomes = ["rock smashes scissors!", "paper covers rock!", "scissors cut paper!", "Tie!"]
+
+    def __init__(self, player_choice, computer_choice):
+        # get attack statement
+        # print(f"player choice: {player_choice}. computer choice {computer_choice} comparison matrix: {self.choices.index(player_choice), self.choices.index(computer_choice)}")
```
Comment (asparageist) [inline mags/RPS.py]: did you intend to leave coding comments?

File: mags/RPS.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        # player choose
+        print('Now the player will choose.')
+        print('\nChoose from the following:')
+        for choice in game_chosen.choices:
+            print(f"{game_chosen.choices.index(choice) + 1}. {choice}")
+        player_choice = False
+        while not player_choice:
+            choice_attempt = input("\nYour choice?  ")
+            try:
+                choice_attempt == game_chosen.choices[int(choice_attempt) -1]
```
Comment (vaelentine) [inline mags/RPS.py]: pycharm complains that "this statement has no effect." Maybe double check this usage

Comment (vaelentine) [inline mags/RPS.py]:
```suggestion
```

File: mags/RPS.py (hunk 1 above)
Comment (vaelentine) [reply mags/RPS.py]: Nope, good catch!

File: mags/RPS.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+# For storing all of the game logic and methods
+class RPS:
+    name = "Rock, Paper, Scissors"
+    choices = ["rock", "paper", "scissors"]
+    win = [0, -1, 1], [1, 0, -1], [-1, 1, 0]
+    outcomes = ["rock smashes scissors!", "paper covers rock!", "scissors cut paper!", "Tie!"]
+
+    def __init__(self, player_choice, computer_choice):
+        # get attack statement
+        # print(f"player choice: {player_choice}. computer choice {computer_choice} comparison matrix: {self.choices.index(player_choice), self.choices.index(computer_choice)}")
```
Comment (vaelentine) [inline mags/RPS.py]:
```suggestion
```

**Review by asparageist — APPROVED** [review]

**Pull request merged**